### ISSUE TITLE
Bootstrap - CRLF line endings in entrypoint.bash leads to failing Cloud Build job due to no such file or directory error

### ISSUE THREAD

**Issue by harrytsee** [terraform-google-modules/terraform-example-foundation#694]
### TL;DR

When deploying the CFT Bootstrap (step 0) locally on a Windows machine to a fresh organisation the following error occurs during the Cloud Build submit step after running the terraform apply:

```
module.cloudbuild_bootstrap.null_resource.cloudbuild_terraform_builder (local-exec): Step #1: Already have image: [europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform](http://europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform)
module.cloudbuild_bootstrap.null_resource.cloudbuild_terraform_builder (local-exec): Step #1: standard_init_linux.go:228: exec user process caused: no such file or directory
module.cloudbuild_bootstrap.null_resource.cloudbuild_terraform_builder (local-exec): Finished Step #1
module.cloudbuild_bootstrap.null_resource.cloudbuild_terraform_builder (local-exec): ERROR
module.cloudbuild_bootstrap.null_resource.cloudbuild_terraform_builder (local-exec): ERROR: build step 1 "[europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform](http://europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform)" failed: step exited with non-zero status: 1
module.cloudbuild_bootstrap.null_resource.cloudbuild_terraform_builder (local-exec): --------------------------------------------------------------------------------
╷
│ Error: local-exec provisioner error
│
│   with module.cloudbuild_bootstrap.null_resource.cloudbuild_terraform_builder,
│   on .terraform\modules\cloudbuild_bootstrap\modules\cloudbuild\[main.tf](http://main.tf/) line 194, in resource "null_resource" "cloudbuild_terraform_builder":
│  194:   provisioner "local-exec" {
│
│ Error running command '      gcloud builds submit .terraform/modules/cloudbuild_bootstrap/modules/cloudbuild/cloudbuild_builder/ --project prj-b-cicd-c727
│ --config=.terraform/modules/cloudbuild_bootstrap/modules/cloudbuild/cloudbuild_builder/cloudbuild.yaml
│ --substitutions=_TERRAFORM_VERSION=0.13.7,_TERRAFORM_VERSION_SHA256SUM=4a52886e019b4fdad2439da5ff43388bbcc6cce9784fde32c53dcd0e28ca9957,_TERRAFORM_VALIDATOR_RELEASE=v0.6.0,_REGION=europe-west2,_REPOSITORY=prj-tf-runners

[...]

│ Step #0: Step 12/12 : ENTRYPOINT ["/builder/entrypoint.bash"]
│ Step #0:  ---> Running in 83aaa6d5192e
│ Step #0: Removing intermediate container 83aaa6d5192e
│ Step #0:  ---> 77e0355a169d
│ Step #0: Successfully built 77e0355a169d
│ Step #0: Successfully tagged [europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform:latest](http://europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform:latest)
│ Finished Step #0
│ Starting Step #1
│ Step #1: Already have image: [europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform](http://europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform)
│ Step #1: standard_init_linux.go:228: exec user process caused: no such file or directory
│ Finished Step #1
│ ERROR
│ ERROR: build step 1 "[europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform](http://europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform)" failed: step exited with non-zero status: 1
│ --------------------------------------------------------------------------------
```

### Solution:
Update entrypoint bash to LF by default or add documentation to troubleshooting steps to update entrypoint.bash CRLF end of line conversion to LF following these [steps](https://stackoverflow.com/questions/51508150/standard-init-linux-go190-exec-user-process-caused-no-such-file-or-directory) for any "no such file or directory" errors.

### Expected behavior

Successful terraform apply following run, this error should not occur as is all steps are executed following instructions and run from an up to date clone of the repo.

### Observed behavior

Key Errors:

```
(local-exec): Step #1: standard_init_linux.go:228: exec user process caused: no such file or directory

│ Error: local-exec provisioner error
│
│   with module.cloudbuild_bootstrap.null_resource.cloudbuild_terraform_builder,
│   on .terraform\modules\cloudbuild_bootstrap\modules\cloudbuild\[main.tf](http://main.tf/) line 194, in resource "null_resource" "cloudbuild_terraform_builder":
│  194:   provisioner "local-exec" {
│
│ Error running command '      gcloud builds submit .terraform/modules/cloudbuild_bootstrap/modules/cloudbuild/cloudbuild_builder/ --project prj-b-cicd-c727
│ --config=.terraform/modules/cloudbuild_bootstrap/modules/cloudbuild/cloudbuild_builder/cloudbuild.yaml
│ --substitutions=_TERRAFORM_VERSION=0.13.7,_TERRAFORM_VERSION_SHA256SUM=4a52886e019b4fdad2439da5ff43388bbcc6cce9784fde32c53dcd0e28ca9957,_TERRAFORM_VALIDATOR_RELEASE=v0.6.0,_REGION=europe-west2,_REPOSITORY=prj-tf-runners

ERROR: build step 1 "[europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform](http://europe-west2-docker.pkg.dev/prj-b-cicd-c727/prj-tf-runners/terraform)" failed: step exited with non-zero status: 1
```

### Terraform Configuration

```hcl
n/a - error stems from entrypoint.bash
```

### Terraform Version

```sh
1.1.7
```

**Comment by daniel-cit** [comment]
Hi @harrytsee Thanks for the report.

Downloading a [zip file](https://github.com/terraform-google-modules/terraform-google-bootstrap/archive/refs/heads/master.zip) with the source code for the [bootstrap](https://github.com/terraform-google-modules/terraform-google-bootstrap) module (the source of the `entrypoint.bash` file) it is possible to check that the line ending used in the file is  `LF`.

Maybe the issue is been caused by the [automatic substitution that git does with the line ending](https://stackoverflow.com/questions/2517190/how-do-i-force-git-to-use-lf-instead-of-crlf-under-windows) of the files in the Windows SO?
